source/extensions/filters/http/ext_proc/mutation_utils.cc

```diff
@@ -151,6 +151,13 @@ absl::Status MutationUtils::applyHeaderMutations(const HeaderMutation& mutation,
       continue;
     }
 
+    // Always skip setting content length header in STREAMED body mode (i.e., `is_streamed_mode` is
+    // true).
+    if (sh.header().key() == "content-length" && is_streamed_mode.has_value() &&
```

Comment:
Maybe use the `ContentLength` constant rather than hardcode. What if there is mixed case?

Reply:
Good point! Done. Also, left a TODO for normalizing the headers to lower-case